vite.config.ts

```diff
@@ -0,0 +1,53 @@
+import { resolve } from "path";
+import { defineConfig, mergeConfig } from "vite";
+import { defineConfig as defineVitestConfig } from "vitest/config";
+import react from "@vitejs/plugin-react";
+
+const vitestConfig = defineVitestConfig({
+    test: {
+        globals: true,
+        include: ["test/**/*.{ts,tsx}"],
+    },
+});
+
+export default mergeConfig(
+    defineConfig({
+        plugins: [react()],
+        build: {
+            lib: {
+                // Could also be a dictionary or array of multiple entry points
+                entry: resolve(__dirname, "src/index.tsx"),
+                name: "PointCloud",
```

Comment:
Does this do anything?

Reply:
I learned today that this was optional, I will remove